test/functional/api/cas/cache.py

```diff
@@ -92,6 +93,19 @@ def get_cache_statistics(self,
                              io_class_id: int = None,
                              stat_filter: List[StatsFilter] = None,
                              percentage_val: bool = False):
+        stats = get_statistics(self.cache_id, None, io_class_id,
+                               stat_filter, percentage_val)
+        if io_class_id is not None:
+            return IoClassStats(stats)
+        return CacheStats(stats)
```

Comment:
Returning objects of two different classes that have no common base from one function isn't good programming practice. Please provide separate function for getting io class stats.

Reply:
done